packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -191,6 +193,11 @@ class PopupMenuItem<T> extends PopupMenuEntry<T> {
   @override
   final double height;
 
+  /// The text style of the entry.
+  ///
+  /// Defaults to [blackMountainView].
```

Comment:
It looks like this doesn't actually default to blackMountainView. You can say something like:

`If this is null, then popupMenuTheme.textStyle is used. If that is also null, then theme.textTheme.subhead is used.`

Reply:
Done